pymc3/distributions/discrete.py

```diff
@@ -1721,18 +1727,30 @@ def dist(cls, eta, cutpoints, *args, **kwargs):
         )
         p = p_cum[..., 1:] - p_cum[..., :-1]
 
-        return super().dist(p, **kwargs)
+        return super().dist(p, *args, **kwargs)
 
 
-class OrderedProbit(Categorical):
+class OrderedLogistic:
```

Comment:
It's probably better to move the docstrings to the user-facing class, or they won't find them.

Reply:
Indeed. That's done